server/src/assetComputeDevTool.js

```diff
@@ -12,17 +12,19 @@
 
 "use strict";
 
-const { AssetComputeClient } = require("@adobe/asset-compute-client");
+const { AssetComputeClient, getIntegrationConfiguration } = require("@adobe/asset-compute-client");
```

Comment:
Would have been nice to have `getIntegrationConfiguration` in a class, even if it would have been a static one.

Reply:
Cleanup JIRA ticket?

Reply:
Ack